lib/rust/parser/debug/tests/parse.rs

```diff
@@ -841,13 +851,13 @@ fn operator_sections() {
 #[test]
 fn template_functions() {
     #[rustfmt::skip]
-    test("_.map (_+2 * 3) _*7", block![
+    test("_.map (_ + 2*3) _*7", block![
```

Comment:
The semantics of this test have changed, because the original syntax can no longer be expressed with template functions.